packages/client/src/navigation/index.ts

```diff
@@ -229,8 +229,10 @@ export function goToTeamSearch(searchedLocation?: searchedLocation) {
 }
 
 export function goToPerformanceHome(
-  timeStart: Date = startOfMonth(subMonths(new Date(Date.now()), 11)),
-  timeEnd: Date = new Date(Date.now()),
+  timeStart: Date = new Date(
+    startOfMonth(subMonths(new Date(Date.now()), 11)).setHours(0, 0, 0, 0)
+  ),
+  timeEnd: Date = new Date(new Date(Date.now()).setHours(0, 0, 0, 0)),
```

Comment:
Would it not be better to use setHours(23, 59, 59)? That way the result will include the updated values for the current day

Reply:
Good spot @Zangetsu101, so only for `timeEnd` it should be like below, right?
```suggestion
  timeEnd: Date = new Date(new Date(Date.now()).setHours(23,59,59)),
```

Reply:
yea this should work right?

Reply:
Yes, it is done now